neurodamus/commands.py

```diff
@@ -55,6 +55,7 @@ def neurodamus(args=None):
         --enable-shm=[ON, OFF]  Enables the use of /dev/shm for coreneuron_input [default: ON]
         --model-stats           Show model stats in CoreNEURON simulations [default: False]
         --dry-run               Dry-run simulation to estimate memory usage [default: False]
+        --skip-write-model      Run CoreNeuron in direct mode, without writing model data to disk
```

Comment:
I'm wondering if this would be better as a `--write-model` and `--no-write-model` pair of options? Would make the `if not skip_write_model` a bit friendlier as `if write_model`.

Otherwise great to see this!

Reply:
There are several options _--do-that=[ON, OFF]_
Maybe this new option should be consistent with this pattern i.e "--write-model=[ON, OFF]"

Reply:
I've renamed the CLI option "--coreneuron-direct-mode".  Does it sound better?